spec/models/cookbook_spec.rb

```diff
@@ -163,6 +163,17 @@
       expect(cookbook.collaborator_users).to_not include(hank)
     end
 
+    it 'should create a new collaborator record for the previous owner' do
+      hank = create(:user)
+      create(:cookbook_collaborator, resourceable: cookbook, user: hank)
+      expect(cookbook.owner).to eql(jimmy)
```

Comment:
Not a fan of having multiple expectations in a single `it` block. Would it be possible to either split this up or do a single data structure comparison?

http://betterspecs.org/#single